src/main/java/com/faforever/api/forum/NodebbService.java

```diff
@@ -0,0 +1,108 @@
+package com.faforever.api.forum;
+
+import com.faforever.api.config.FafApiProperties;
+import com.faforever.api.user.UserDataSyncService;
+import com.faforever.api.user.UserUpdatedEvent;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Optional;
+
+@Service
+@ConditionalOnProperty(value = "faf-api.nodebb.master-token")
```

Comment:
So does Spring not create this bean if that  value is set to "false"

Reply:
Yes.